app/src/main/res/values/settings.xml

```diff
@@ -36,6 +36,24 @@
         <item>@string/settings_brown_theme</item>
     </string-array>
 
+    <string-array name="column_count_labels">
+        <item>@string/settings_label_default_column_count</item>
+        <item>@string/settings_label_column_count_1</item>
+        <item>@string/settings_label_column_count_2</item>
+        <item>@string/settings_label_column_count_3</item>
+        <item>@string/settings_label_column_count_4</item>
+        <item>@string/settings_label_column_count_5</item>
+    </string-array>
+
+    <string-array name="column_count_values">
+        <item>@string/settings_symbol_default_column_count</item>
+        <item>1</item>
+        <item>2</item>
+        <item>3</item>
+        <item>4</item>
+        <item>5</item>
+    </string-array>
```

Comment:
Please be consistent in naming with the other string arrays (they use `x_values` and `x_value_strings` and `@string/_ values` for `strings.xml` (they use variable names like `@string/settings_key_x` and `@string/settings_x`)